File: C++ programs/armstrong.cpp
#include<iostream>
using namespace std;
int main()
{
	int rem,n,digit=0,og;
	cout<<"Enter any number:";
	cin>>n;
	og=n;
	while(n!=0)
	{
		rem=n%10;
		digit+=rem*rem*rem;
		n/=10;
	}
	if(og==digit)
	{
		cout<<"This is an armstrong number";
	}
	else
	{
		cout<<"This is not an armstrong number";
	}
}

File: python/armstrong.py
n = int(input("Enter any no: "))
sum = 0
temp = n
while temp > 0:
    digit = temp % 10  # 3   5
    sum = sum + (digit ** 3)  # 0+27=27  125
    temp //= 10  # n=15
print(sum)
if n == sum:
    print("It is Armstrong no.")
else:
    print("It is not an Armstrong no.")
